docs/help.rst

```diff
@@ -0,0 +1,5 @@
+****
+Help
+****
+
+For questions and/or assistance with STIPS, please contact the STIPS Helpdesk via email at help@stsci.edu, with the subject line "STIPS Question".
```

Comment:
Do we want to steer people toward Service Now, too? That interface has a few more features than are available over email alone.

Reply:
I don't know. I had thought that we had gotten feedback from the Roman Mission Office that we wanted to only provide the e-mail address, at least for the moment, but I can't find an actual note of that anywhere.

Reply:
Gisella says the email address is still the one to emphasize.